presto-tests/src/test/java/com/facebook/presto/utils/QueryExecutionClientUtil.java

```diff
@@ -148,4 +149,10 @@ public static QueryStateInfo getQueryStateInfo(HttpClient client, TestingPrestoS
         Request request = prepareGet().setUri(server.resolve(path)).build();
         return client.execute(request, createJsonResponseHandler(jsonCodec(QueryStateInfo.class)));
     }
+
+    public static TaskInfo getTaskInfo(HttpClient client, TestingPrestoServer server, String path)
+    {
+        Request request = prepareGet().setUri(server.resolve(path)).build();
+        return client.execute(request, createJsonResponseHandler(jsonCodec(TaskInfo.class)));
+    }
```

Comment:
I would reuse getResponseEntity instead of duplicating code here and above.

Reply:
Removed this portion and reused `getResponseEntity`